test/unit/filter-ignore.test.ts

```diff
@@ -248,3 +248,30 @@ test('filters vulnerabilities by exact match', async () => {
   const filtered = config.filter(vulns, undefined, 'exact');
   expect(filtered.vulnerabilities).toStrictEqual(expected.vulnerabilities);
 });
+
+test('vulnerabilities filter is case insensitive', async () => {
+  const vulns = {
+    vulnerabilities: [
+      {
+        id: 'A-vuLn',
+        from: ['dir/file.json', 'foo', 'bar'],
+      },
+      {
+        id: 'A-VULN',
+        from: ['file.json', 'foo', 'bar'],
+      },
+      {
+        id: 'another-vuln',
+        from: ['file.json', 'foo', 'bar'],
+      },
+    ],
+  } as VulnerabilityReport;
+
+  const expected = cloneDeep(vulns);
+  expected.vulnerabilities.splice(1, 1);
+
+  const config = await policy.load(__dirname + '/../fixtures/ignore-exact');
+
+  const filtered = config.filter(vulns, undefined, 'exact');
+  expect(filtered.vulnerabilities).toStrictEqual(expected.vulnerabilities);
+});
```

Comment:
Do you think we could be more explicit with the **expected result** and the `vundId` being tested? Maybe then we don't need to mutate `expected.vulnerabilities` with the `splice` API?

Is it correct to say that the expected result is:

```ts
  const expected = {
    vulnerabilities: [
      {
        id: 'A-vuLn',
        from: ['dir/file.json', 'foo', 'bar'],
      },
      {
        id: 'another-vuln',
        from: ['file.json', 'foo', 'bar'],
      },
    ],
  } as VulnerabilityReport;
```

My suggestion is more about seeing clearly that the filter is filtering out the proper ID instead of knowing that the array started with three items and resulted in 2.

WDYT?

Reply:
Sounds good to me! Pushed an update.